include/RandLAPACK/lapackpp.h

```diff
@@ -0,0 +1,12 @@
+#ifndef randlapack_lapackpp_h
+#define randlapack_lapackpp_h
+
+#define LAPACK_COMPLEX_CPP 1
```

Comment:
What is this define statement for?

Reply:
it tells LAPACK++ not to `#include <complex.h>`, the C header. Google test needs the C++ version in `#include <complex>`.

Reply:
Gotcha. Thanks!